### PR TITLE
AR Wintermute and more

### PR DESCRIPTION
**Pull request by RandomColonist**
- Remade BR into AR, remade mags, desc, reduced size to large from huge, reduced price from 4000 to 3500 (regular AR), added zoom because it has scope, removed DMG multiplier, replaced 250 RPM mode with 400 RPM, added "short burst" mode, replaced old 7.62 mags in secure armoury with new IH 5.56 mags.
- Fixed shitton of bullshit in LMG code, removed unused vars, altered used vars, altered file structure to make it more logical.
- Made PK boxes into mag boxes to work with magwells.
- c20r no more  inherits all possible fire modes for automatic weapons, now it has 400RPM full auto, semi-auto fire with no move delay, and 3-round burst.
- Altered Z8 Bulldog description, moved "space marine" meme to Dallas where it belongs.
- STS-35 lost illegal tech levels because why would it have it? Anyone can get those gun, there's nothing special in it.
- Reduced 6.5 mm bullet AP from 20 to 15, because it was literally almost good as 5.56 IH rifle.
- Altered Molly description to let players know that it accepts both types of mags.

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/automatic/z8.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/projectile/automatic/z8
 	name = "FS AR 5.56x45mm \"Z8 Bulldog\""
-	desc = "The Z8 Bulldog is an older model bullpup carbine, made by \"Frozen Star\". Uses armor piercing 5.56mm rounds. Makes you feel like a space marine when you hold it."
+	desc = "The Z8 Bulldog is an older model bullpup carbine, made by \"Frozen Star\". Uses 5.56mm rounds."
```
Comment (ACCount12) [inline code/modules/projectiles/guns/projectile/automatic/z8.dm]: Why remove that line?

File: code/modules/projectiles/guns/projectile/automatic/IH_heavy.dm [hunk 2]
```diff
@@ -1,28 +1,30 @@
-//The heavy rifle is ironhammer's go-to weapon for dealing with serious threats. It weighs a ton and hits hard
+//Primary AR of IH. Has supreme AP to deal with those pesky Serbs (guildsmans actually) in armored hardsuits.
```
Comment (seekin4u) [inline code/modules/projectiles/guns/projectile/automatic/IH_heavy.dm]: что значат все эти сокращения? типо пограмистер должен думать не только как написать код, а еще и над тем как это все расшифровать?

Comment (RandomColonist) [reply code/modules/projectiles/guns/projectile/automatic/IH_heavy.dm]: Хватит тут срать анрелейтед комментариями на русском.
AR - Assault Rifle
IH - Ironhammer
Ничего такого не использовал, ну.

File: code/modules/projectiles/guns/projectile/automatic/z8.dm (hunk 1 above)
Comment (RandomColonist) [reply code/modules/projectiles/guns/projectile/automatic/z8.dm]: We don't have AP 5.56, only generic ones. Yes, they have higher AP, but you can't notice that even when comparing to AK. And last line moved to Dallas rifle, true weapon of space bald marine.

**Comment by ACCount12** [comment]
Conflict central. Ready to merge once you resolve this mess.

**Pull request merged**